frontend/src/pages/modelCatalog/screens/ModelCatalog.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+
+import ApplicationsPage from '~/pages/ApplicationsPage';
+import TitleWithIcon from '~/concepts/design/TitleWithIcon';
+import { ProjectObjectType, typedEmptyImage } from '~/concepts/design/utils';
+import { conditionalArea, SupportedArea } from '~/concepts/areas';
+import EmptyModelCatalogState from '~/pages/modelCatalog/EmptyModelCatalogState';
+
+const ModelCatalog: React.FC = conditionalArea(
+  SupportedArea.MODEL_CATALOG,
+  true,
+)(() => {
+  const renderStateProps = {
+    empty: true,
+    emptyStatePage: (
+      <EmptyModelCatalogState
+        testid="empty-model-catalog-state"
+        title="Request access to model catalog"
+        description="To request access to model catalog, contact your administrator."
+        headerIcon={() => <img src={typedEmptyImage(ProjectObjectType.registeredModels)} alt="" />}
+      />
+    ),
+    headerContent: null,
+  };
+
+  return (
+    <ApplicationsPage
+      title={
+        <TitleWithIcon title="Model Catalog" objectType={ProjectObjectType.registeredModels} />
+      }
+      {...renderStateProps}
+      loaded
+      provideChildrenPadding
+    />
+  );
+});
```

Comment:
I think we can also just put these `renderStateProps` inline when we return `<ApplicationsPage`, but maybe we leave this alone so we can unblock @gitdallas for implementing this page in https://issues.redhat.com/browse/RHOAIENG-18956 and he can render this however makes sense for that story.

Reply:
yeah, I think Dallas will replace this whole component here with the model catalog page that he will create as part of RHOAIENG-18956. Thats the reason, I just duplicated the content of core loader component here.